src/Whoops/Handler/XmlResponseHandler.php

```diff
@@ -67,7 +67,7 @@ public function contentType()
      */
     private static function addDataToNode(\SimpleXMLElement $node, $data)
     {
-        assert(is_array($data) || $data instanceof Traversable);
+        assert(is_array($data) || $data instanceof \Traversable);
```

Comment:
Stylistically preferable to import this class instead.

Reply:
Importing...

Reply:
Yeh, I'd say so.